app/Services/Northstar/Northstar.php

```diff
@@ -21,14 +21,14 @@ public function __construct()
     }
 
     /**
-     * Get users from Northstar for seeding database.
+     * Send a GET request to return all users matching a given query from Northstar.
      *
-     * @param  string $limit
-     * @return object|null
+     * @param  array $inputs - Filter, search, limit or pagination queries
```

Comment:
haha, that crossed my mind and I was like... meh, too much work. I'll add it and then merge ;)